target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
I'm not very familiar with this stuff, but - since these are generated files, should we have them in the .gitignore instead of committing them? Same with the maven-status/ files and the .war file

Reply:
i have no idea, I didnt know if they were important or not so i included them in the commit

Reply:
OK - usually we should exclude generated files from the committed repository since they're derived from your code and not actually written by you. It's also good practice because subsequent changes to the code will often affect the generated files, and it's annoying to have to keep re-committing and re-reviewing them when they're functionally meaningless. But since we're not sure, let's leave them in for now and worry about it later 😄